plugin/notes/plugin.js

```diff
@@ -146,7 +146,7 @@ const Plugin = () => {
 		}
 
 		// Look for notes defined in an aside element
-		if( notesElements ) {
+		if( notesElements && notesElements.length ) {
```

Comment:
`notesElements` is either `null`(if there is `.current-fragment`) or a list of elements(result of `slideElement.querySelectorAll( 'aside.notes' )`). 

So if we have a slide:
1. without fragments at all(so no `.current-fragment`)
2. with notes specified through `data-notes`

We did never see the notes because condition `if( notesElements )` was always `true` even though list was empty